src/Cli/dotnet/commands/dotnet-list/dotnet-list-package/ListPackageReferencesCommandParser.cs

```diff
@@ -63,6 +63,9 @@ internal static class ListPackageReferencesCommandParser
         public static readonly Option FormatOption = new ForwardedOption<ReportOutputFormat>("--format", LocalizableStrings.CmdFormatDescription)
         { }.ForwardAsSingle(o => $"--format:{o}");
 
+        public static readonly Option OutputVersiontOption = new ForwardedOption<int>("--output-version", LocalizableStrings.CmdOutputVersionDescription)
```

Comment:
@baronfel 
It's currently just `1` or nothing. I found it's bit challenging to add `1` as enum without changing other codes on how enum is parsed. So, I'm just using int here.